File: new_server/vueblog-common/src/main/java/cn/smilex/vueblog/common/entity/common/Triplet.java
package cn.smilex.vueblog.common.entity.common;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author smilex
 * @date 2022/12/6/7:55
 * @since 1.0
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Triplet<L, M, R> {
    private L left;
    private M middle;
    private R right;
}

File: new_server/vueblog-common/src/main/java/cn/smilex/vueblog/common/entity/common/Triplet.kt
package cn.smilex.vueblog.common.entity.common

/**
 * TODO
 * @author smilex
 * @date 2023/12/4 21:58:50
 */
data class Triplet<L, M, R>(
    var left: L? = null,
    var middle: M? = null,
    var right: R? = null
)